npm install fails

```
❯ npm i
npm WARN deprecated jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated graceful-fs@2.0.3: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.

> fsevents@1.0.11 install /Users/zeke/electron/devtron/node_modules/fsevents
> node-pre-gyp install --fallback-to-build

[fsevents] Success: "/Users/zeke/electron/devtron/node_modules/fsevents/lib/binding/Release/node-v47-darwin-x64/fse.node" is installed via remote

> devtron@1.0.0 install /Users/zeke/electron/devtron
> browserify lib/*.js -o out/index.js --ignore-missing --entry lib/index.js

events.js:154
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open 'out/index.js'
    at Error (native)

npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "i"
npm ERR! node v5.8.0
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE
npm ERR! devtron@1.0.0 install: `browserify lib/*.js -o out/index.js --ignore-missing --entry lib/index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the devtron@1.0.0 install script 'browserify lib/*.js -o out/index.js --ignore-missing --entry lib/index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the devtron package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     browserify lib/*.js -o out/index.js --ignore-missing --entry lib/index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs devtron
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls devtron
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/zeke/electron/devtron/npm-debug.log

~/electron/devtron move-to-electron-org
❯ node -v
v5.8.0

~/electron/devtron move-to-electron-org
❯ npm -v
3.7.3
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,2 +1,1 @@
 /node_modules/
-/out/

--- a/out/.gitignore
+++ b/out/.gitignore
@@ -1,0 +1,2 @@
+*
+!.gitignore

